### PR TITLE
MODORDSTOR-363. Add indexes to improve Get Order/Get Po Line/Open Order/Receive order flow

### PR DESCRIPTION
**Pull request by SerhiiNosko**
## Purpose
https://issues.folio.org/browse/MODORDSTOR-363

### CODE REVIEW

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -200,6 +200,13 @@
         {
           "fieldName": "acqUnitIds"
         }
+      ],
+      "ginIndex": [
+        {
+          "fieldName": "poNumber",
+          "caseSensitive": false,
+          "removeAccents": true
+        }
```
Comment (SerhiiNosko) [inline src/main/resources/templates/db_scripts/schema.json:209]: Resolves these 2 WARN logs:
`14:46:12 [543648/orders;980086/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing LIKE search without index for purchase_order.jsonb->>'poNumber', CQL >>> SQL: poNumber == 10011 >>> lower(f_unaccent(purchase_order.jsonb->>'poNumber')) LIKE lower(f_unaccent('10011'))
`


`10:34:20 [084878/orders;442135/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing wildcard LIKE search without index for purchase_order.jsonb->>'poNumber', CQL >>> SQL: purchaseOrder.poNumber == *10004* >>> lower(f_unaccent(purchase_order.jsonb->>'poNumber')) LIKE lower(f_unaccent('%10004%'))
`

File: src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -430,6 +437,17 @@
...
+      "index": [
+        {
+          "fieldName": "acqUnitIds",
+          "caseSensitive": false
+        }
+      ],
+      "fullTextIndex": [
+        {
+          "fieldName": "acqUnitIds"
+        }
```
Comment (SerhiiNosko) [inline src/main/resources/templates/db_scripts/schema.json:450]: The same configuration of indexes for acq units as for Purchase Order, resolves these 2 WARN logs:
`10:29:15 [815929/orders;164360/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing FT search without index for titles.jsonb->>'acqUnitIds', CQL >>> SQL: acqUnitIds = 8535fa1d-3d86-49cf-84ba-df96cd319c2b >>> get_tsvector(f_unaccent(titles.jsonb->>'acqUnitIds')) @@ tsquery_phrase(f_unaccent('8535fa1d-3d86-49cf-84ba-df96cd319c2b'))
` 


`16:48:41 [797007/orders;863055/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing LIKE search without index for titles.jsonb->>'acqUnitIds', CQL >>> SQL: acqUnitIds <> [] >>> lower(f_unaccent(titles.jsonb->>'acqUnitIds')) NOT LIKE lower(f_unaccent('[]'))
`

File: src/main/resources/templates/db_scripts/schema.json [hunk 3]
```diff
@@ -612,6 +634,13 @@
           "fieldName": "exportJobId",
           "caseSensitive": false
         }
+      ],
+      "ginIndex": [
+        {
+          "fieldName": "exportedPoLineIds",
+          "caseSensitive": false,
+          "removeAccents": true
+        }
```
Comment (SerhiiNosko) [inline src/main/resources/templates/db_scripts/schema.json:643]: Resolves this WARN log:
`10:28:55 [924539/orders;230710/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing wildcard LIKE search without index for export_history.jsonb->>'exportedPoLineIds', CQL >>> SQL: exportedPoLineIds == "*\"293fc302-d1be-407b-a6f8-292fa7cb7d91\"*" >>> lower(f_unaccent(export_history.jsonb->>'exportedPoLineIds')) LIKE lower(f_unaccent('%"293fc302-d1be-407b-a6f8-292fa7cb7d91"%'))
`

File: src/main/resources/templates/db_scripts/schema.json [hunk 4]
```diff
@@ -612,6 +634,13 @@
...
       ]
```
Comment (SerhiiNosko) [inline src/main/resources/templates/db_scripts/schema.json:644]: Additional WARN logs for Create Order, Create PO Line, Get Order, Get PO Line, Open Order, Receive Title will be covered by correcting UI queries(3 places to improve) in scope of this story:
https://issues.folio.org/browse/UIOR-1181

**Review by Dmitriy-Butramyou — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**